src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -1536,6 +1539,24 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 value = ASR::down_cast<ASR::expr_t>(ASR::make_ConstantComplex_t(al, loc,
                         std::real(result), std::imag(result), dest_type));
             }
+            else if (ASRUtils::is_logical(*dest_type)) {
+                int8_t left_value = ASR::down_cast<ASR::ConstantLogical_t>(
+                                                    ASRUtils::expr_value(left))->m_value;
+                int8_t right_value = ASR::down_cast<ASR::ConstantLogical_t>(
+                                                    ASRUtils::expr_value(right))->m_value;
+                int8_t result;
+                switch (op) {
+                    case (ASR::binopType::Add): { result = left_value + right_value; break; }
+                    case (ASR::binopType::Sub): { result = left_value - right_value; break; }
+                    case (ASR::binopType::Mul): { result = left_value * right_value; break; }
+                    case (ASR::binopType::Pow): { result = std::pow(left_value, right_value); break; }
+                    case (ASR::binopType::Div): { } // TODO: Handle division of logicals
```

Comment:
I learned something new, I didn't know you can do this in Python, but I guess you can:
```
In [3]: True + True
Out[3]: 2

In [4]: True * True
Out[4]: 1

In [5]: True / True
Out[5]: 1.0
```

Reply:
Yes.
```python
>>> isinstance(True, int)
True
```